### ISSUE TITLE
Can't select checkboxes in block display if width less than 100%

### ISSUE THREAD

**Issue by tsd-development** [elliotcondon/acf#458]
![screen shot 2015-10-12 at 4 58 36 pm](https://cloud.githubusercontent.com/assets/10913187/10438639/c4e67ffc-7102-11e5-88ae-558260efa423.png)

Howdy! Long time user, first time poster. First off, let me thank you for creating such a great project that really makes my job... well... possible. Huge, huge kudos.

I've got an issue here that looks like it could be a pretty simple thing to solve. I've got a few switches at the top of a flexible content field. They're serving as switches that activate certain content blocks. After the switches, I've got a flexible content field. The label for the flexible content field isn't clearing the row of switches (which are all display:block, width: 33%).

I feel like a rule with a clear statement would help out - just not sure where to put it.

Thanks!!

Gregg

**Comment by elliotcondon** [comment]
Hi @tsd-development 

Thanks for the bug report.

Which version of ACF are you using?
Is it possible that you have a field above your flexible content field which is hidden by conditional logic?

**Comment by ghurson** [comment]
I do! For a quick fix, I've put a message field directly after the three fields.

**Comment by elliotcondon** [comment]
Thanks @ghurson 

I've also had a similar bug report and will have it fixed in the next version.

For now, please add the following CSS to the admin:

```
.acf-fields > .acf-field {
	clear: left; /* used to clear floating widths */
}
```

**Comment by tsd-development** [comment]
Thanks so much! Cheers :)

On Tue, Oct 13, 2015 at 6:53 PM, Elliot Condon <notifications@github.com>
wrote:

> Closed #458 <https://github.com/elliotcondon/acf/issues/458>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/elliotcondon/acf/issues/458#event-434624423>.
>



-- 
Gregg Hurson
Top Shelf Design, LLC

*define.create.deliver. *

1011 Arlington Boulevard Suite 330
Arlington, VA 22209
571-431-6101 x. 106 Office
202-595-9161 Fax
gregg@topshelfdesign.net
www.topshelfdesign.net
